onnxmltools/convert/common/_container.py

```diff
@@ -80,6 +80,14 @@ def add_output(self, variable):
             self._outputs.append(variable)
 
 
+class SklearnModelContainer(CommonSklearnModelContainer):
```

Comment:
If there is no any difference between sklearn and lightbgm model containter,  can we just rename SklearnModelContainer as SklearnAndLightGBMModelContainer or a better name?